### ISSUE TITLE
Google Maps - data layer partially disappears

### ISSUE THREAD

**Issue by hughsheehy** [visgl/deck.gl#5927]
#### Description
If I click on a datapoint on the right side of the map (only the right side) and open a tooltip, the data layer partially disappears. The whole google map also seems to jump. 

See video here. https://www.youtube.com/watch?v=htOPvZTeVYI 

The amount of the data layer that vanishes seems (I think) to be related to the width of the infowindow that's opened. 

#### Expected Behavior
That the tooltip should open on click and nothing else. 

#### Repro Steps
Go to https://toolkit.goeve.co.uk/map.html  and click on any charge point at the right edge of the screen. 

Happens on Chrome, Firefox, and Edge. Also chrome on Android.   All code is in the page (pure JS, no framework). 

#### Environment

Deck gl 7.0.9
Happens on all desktop browsers on PC, also on Chrome on Android. 

 - Framework Version: [e.g. deck.gl 8.3.0 / pydeck 0.5.0]
 - Browser Version: [e.g. Chrome 78.0]
 - OS: [e.g. Mac OS X 10.15 / iOS 14.2]

#### Logs
No error messages appear.

**Comment by Pessimistress** [comment]
I don't think this is related to deck.gl. If you look at the navigation control buttons and the attribution on the bottom right, they shift when your info window is open. The deck.gl canvas remain synchronized with the map, and the map itself is not positioned correctly.

**Comment by hughsheehy** [comment]
> I don't think this is related to deck.gl. If you look at the navigation control buttons and the attribution on the bottom right, they shift when your info window is open. The deck.gl canvas remain synchronized with the map, and the map itself is not positioned correctly.

I guess that's possible. But I've made many many google maps over the years and never had this before.

**Comment by Pessimistress** [comment]
Here is a stripped down version of your page: https://codepen.io/Pessimistress/pen/VwbYgvq

Click anywhere and you can see the entire map shifts to the left.

**Comment by hughsheehy** [comment]
Thanks. Sorry to bother you with what wasn't a bug!
I don't actually understand what the bug was in the google map, In any case, I've upgraded to the newest Deck version and rearranged the whole map so that the bug doesn't seem to happen any more anyway. I followed one of your (older) patterns for the tooltips - which works really nicely for the purpose. I'll look at Deck's onClinck and onHover today. Usability on mobile is a big priority for us, which is one of the reasons the older (simpler) pattern works so nicely.
